jgrapht-core/src/test/java/org/jgrapht/alg/scoring/AlphaCentralityTest.java

```diff
@@ -0,0 +1,353 @@
+/*
+ * (C) Copyright 2018-2018, by Pratik Tibrewal and Contributors.
+ *
+ * JGraphT : a free Java graph-theory library
+ *
+ * This program and the accompanying materials are dual-licensed under
+ * either
+ *
+ * (a) the terms of the GNU Lesser General Public License version 2.1
+ * as published by the Free Software Foundation, or (at your option) any
+ * later version.
+ *
+ * or (per the licensee's choosing)
+ *
+ * (b) the terms of the Eclipse Public License v1.0 as published by
+ * the Eclipse Foundation.
+ */
+package org.jgrapht.alg.scoring;
+
+import java.util.*;
+import java.util.function.ToDoubleFunction;
+import org.jgrapht.alg.interfaces.*;
+import org.jgrapht.graph.*;
+
+import org.junit.*;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Unit tests for AlphaCentrality
+ * 
+ * @author Pratik Tibrewal
+ */
+public class AlphaCentralityTest
+{
+	@Test
+    public void testGraph2Nodes()
+    {
+        DirectedPseudograph<String, DefaultEdge> g = new DirectedPseudograph<>(DefaultEdge.class);
+
+        g.addVertex("1");
+        g.addVertex("2");
+        g.addEdge("1", "2");
+        g.addEdge("2", "1");
+
+        VertexScoringAlgorithm<String, Double> pr = new AlphaCentrality<>(g);
+
+        // assertEquals(pr.getVertexScore("1"), pr.getVertexScore("2"), 0.0001);
```

Comment:
Why is this commented out?

Reply:
Fixed! Moreover @jsichi do you recommend squashing these commits or is it fine? I think squashing will be good right?

Reply:
Yes, squash please.

Reply:
Done.